docs/API/LorisRESTAPI.md

```diff
@@ -1,4 +1,4 @@
-# Loris API - v0.0.2
+# Loris API - v0.0.3-dev
```

Comment:
I think this change it meant to go in the file https://github.com/xlecours/Loris/blob/58750230e6618886030d9f00d681454ca9149a01/docs/API/LorisRESTAPI_v0.0.3.md in the tree, not the stable doc?